### DISCUSSION TITLE
how to setup acme provisioner with step ca and  a other maxTLSCertDuration

### DISCUSSION THREAD

**Discussion by huckley** [smallstep/certificates#987] [Q&A] [answered]
I use the docker container and i search I a howto to setup a acme provisioner with a custom maxTLSCertDuration using the step cli.
step cli version 0.19.0 do not know the `step ca provisioner update` command.

**Comment by tashian** [comment, marked as answer]
That's correct. You'll need to upgrade to `step` 0.21.0 or higher to get support for the [remote provisioner management](https://smallstep.com/docs/step-ca/provisioners/#remote-provisioner-management) commands.
